icons/Settings.icon.tsx

```diff
@@ -4,7 +4,7 @@ import Svg, { Path } from 'react-native-svg';
 export const SettingsIcon = () => {
   return (
     <>
-      <Svg width="24" height="24" viewBox="0 0 24 24" fill="none">
+      <Svg width="26" height="26" viewBox="0 0 24 24" fill="none">
```

Comment:
Same as above, check if this renders as expected in both android and ios builds.